src/ShareButton.css

```diff
@@ -0,0 +1,5 @@
+.ShareButton {
+    margin-top: 1rem;
+    height: 2.25rem;
```

Comment:
I think we don't need the `margin-top` and `height` properties here.

Reply:
Removed.